sites/platform/src/dedicated-gen-2/architecture/options.md

```diff
@@ -30,12 +33,13 @@ The cluster begins to look more like a standard N-Tier architecture at this poin
 Speak to your sales representative about the costs associated with adding additional application servers. 
 This configuration requires a separate setup from the default so advanced planning is required.
 
-## SFTP accounts
+## `sftp`
 
-In addition to SSH accounts, SFTP accounts can be created with a custom user/password that are restricted to certain directories. 
-These directories must be one of the writeable mounts (or rather, there’s no point assigning them to the read-only code directory). 
-There is no cost for this configuration, and it can be requested at any time via a support ticket. 
-SSH public key based authentication is also supported on the SFTP account.
+In addition to SSH accounts, `sftp` accounts can be created with a custom user/password that are restricted to certain directories. 
```

Comment:
really? TIL. and that would be incredibly useful on the grid!

Reply:
Should we link from this section back over to /development/file-transfer.html#sftp ?

Reply:
Absolutely!